docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,183 @@
+    Your browser does not support the audio element.
+    </audio>
+
+Disarming
+------------
+If you are using the :ref:`Advanced Arming feature <manual_advanced_arming>`, then, on disarming, your module will perform its :ref:`advanced_disarming_behavior`. 
+If the song playback option is configured to either Once or Continuously, you will hear the disarming song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/disarm.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If the song playback option is set to Continuously, the disarm song will repeat until the disarm state is broken.
+
+Analog Hobby Protocol Calibration
+-------------------------------------
+While :ref:`performing analog hobby protocol calibration <hobby_calibration_tutorial>`, you will hear two separate song portions indicating the calibration's progress. 
+The first portion of the song indicates that calibration is starting. This is followed by tones played twice as fast indicating that calibration is in progress. 
+An example of a full calibration song is played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/hobby_calibration.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Timeout
+----------
+When a :ref:`timeout <manual_timeout>` is reached, your module will play the timeout song as specified by its :ref:`timeout_behavior`. 
+When the Song Playback is set to play either once or continuously, you will here the timeout song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/timeout.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If the song playback option is set to Continuously, the timeout song will repeat until the timeout is ended.
+
+Watchdog
+-----------
+Watchdog timers are used to indicate that there has been a blocking error in the firmware. When a watchdog error occurs, the module restarts, and plays the watchdog error song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/watchdog.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If your module plays the watchdog error song, please reach out to our support team at support@vertiq.co with the details surrounding the watchdog. 
+Any flight logs, support files, etc. that can be provided will help our team best diagnose and fix the issue in a new firmware release.
+
+Buzzer Song Volume
+====================
+If the audio feedback from your module is either too loud or quiet, you can adjust its volume with the Volume Max parameter available in the Control Center's advanced tab.
```

Comment:
Do we typically capitalize the names of tabs like Advanced? If so, should capitalize it here